honestcash-v2/src/app/app.component.ts

```diff
@@ -15,49 +10,11 @@ export class AppComponent implements OnInit {
   @HostBinding('class') class = 'h-full w-full flex flex-wrap';
   title = 'honestcash-v2';
 
-  walletState: Observable<any>;
-  authState: Observable<any>;
-
   constructor(
-    private store: Store<AppState>,
     private authService: AuthService
-  ) {
-    this.walletState = this.store.select(selectWalletState);
-    this.authState = this.store.select(selectAuthState);
-  }
+  ) {}
 
   ngOnInit() {
-    this.setupUser();
-    this.setupWallet();
-  }
```

Comment:
The functionality needs to be there.

- On the start of the application we need to check if the user is logged in.
- On the start of the application we need to init the wallet.

Reply:
If you are logged in, you should be forwarded to the `/` (the feed page)